src/main/webapp/dev/js/main/adminSearch.js

```diff
@@ -144,4 +144,60 @@ $(document).ready(() => {
+        const sessionStatus = $entry.find('input[name=sessionStatus]').val();
+        const sessionName = $entry.find('#feedback-session-name').text();
+        const sessionNameWithoutDate = sessionName.substring(0, sessionName.lastIndexOf(' ['));
+
+        const relatedLink = encodeURIComponent($entry.find('input[name=relatedLink]').val());
+        const googleSignup = encodeURIComponent('https://accounts.google.com/NewAccount');
+
+        const bodyHeader = encodeURI(`Hello ${studentName},`);
+        const bodyFooter =
+                encodeURI(`${crlf}${crlf}If you encounter any problems when using the system,`
+                + ` you can email TEAMMATES support team at ${supportEmail}.`
+                + `${crlf}${crlf}Regards,${crlf}TEAMMATES Team.`);
+
+        const uniqueLinkMessage = 'The above link is unique to you. Please do not share it with your classmates.';
+
+        let subject = '';
+        let bodyContent = '';
+
+        if (subjectType === 'Invitation to join course') {
+            subject = encodeURI(`TEAMMATES: ${subjectType} [${courseName}][Course ID: ${courseId}]`);
+            bodyContent = encodeURI(
+                    `${crlf}${crlf}The course ${courseName} is using the TEAMMATES System to collect feedback.`
+                    + ` ${crlf}${crlf}To 'join' the course, please go to this Web address: ${relatedLink}`
+                    + ` ${crlf}${crlf}*If prompted to log in, use your Google account to log in. If you do not have`
+                    + ` a Google account, please create one from the Google Accounts page: ${googleSignup}`
+                    + ` ${crlf}*${uniqueLinkMessage} ${crlf}${crlf}Note that If you wish to access TEAMMATES`
+                    + ' without using your Google account, you do not need to \'join\' the course as instructed above.'
+                    + ' You will still be able to submit/view feedback by following the instructions sent to you by'
+                    + ' TEAMMATES at the appropriate times. However, we recommend joining the course using your Google'
+                    + ' account, because it gives you more convenient access to all your feedback stored in TEAMMATES.');
+        } else {
+            subject =
+                    encodeURI(
+                        `TEAMMATES: ${subjectType} [Course: ${courseName}][Feedback Session: ${sessionNameWithoutDate}]`
+                        );
+            bodyContent =
+                    encodeURI(`${crlf}${crlf}The following feedback session is ${sessionStatus}.${crlf}`
+                    + `Course: [${courseId}][${courseName}]${crlf}Feedback Session Name: ${sessionName}${crlf}`
+                    + `${crlf}The link to the feedback for the above session, please go to this Web`
+                    + ` address: ${relatedLink} ${crlf}${crlf}*${uniqueLinkMessage}`);
+        }
+
+        const emailWrapper = `mailto:${studentEmail}?Subject=${subject}&body=${bodyHeader}${bodyContent}${bodyFooter}`;
```

Comment:
The current implementation works, but why not just do
```JavaScript
const emailWrapper = `mailto:${studentEmail}?Subject=${encodeURI(subject)}&body=${encodeURI(bodyHeader + bodyContent + bodyFooter)}`;
```

This way, you don't have to repeat `encodeURI` so many times, and it's immediately clear from this one line alone that the resulting string is safe, without having to look at the other lines of code.